lib/util/parse_helper.dart

```diff
@@ -0,0 +1,16 @@
+ParseHelper parseHelper = ParseHelper();
+
+class ParseHelper {
+  parseDouble(dynamic value) {
+    if (value is double) return value;
+    if (value is int) return value.toDouble();
+    return null;
```

Comment:
I think this should throw an Exception, otherwise it will come later down the line and is less debuggable